content/docs/welcome-guide/setup/setup-from-github/building-linux.md

```diff
@@ -103,10 +103,11 @@ To prepare to build the engine and projects, choose one of the following build t
 1. Use CMake to build the engine as an SDK, the same as if you installed the engine from an installer tool. The following example shows the `profile` build configuration.
 
     ```shell
-    cmake --build build/linux --target INSTALL --config profile -j <number of parallel build tasks>
+    cmake --build build/linux --target INSTALL --config <debug|profile|release> -j <number of parallel build tasks>
```

Comment:
I think we should keep the example for `profile`. I understand the purpose is to make it clear that O3DE supports different configurations, but the configuration recommended for this workflow is `profile`, and as a "Getting Started" topic, that's most likely enough info for newcomers to know. 

I very much agree that knowing the different configurations are useful, and that information is difficult to find. So in my next comments, I make some suggestions to address that. 
```suggestion
    cmake --build build/linux --target INSTALL --config profile -j <number of parallel build tasks>
```